src/docs/reference/scale-to-zero.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Scale to Zero
+---
+
```

Comment:
I'd add the Priority Boarding banner here, as it's a beta feature. E.g.:
<img width="726" alt="ss 2023-10-05 at 11 37 00 PM" src="https://github.com/railwayapp/docs/assets/24421625/6e8ba9e2-9700-40c9-92ef-d2b7dc9c59c7">